test/jdk/java/awt/PopupMenu/PopupMenuVisuals.java

```diff
@@ -47,7 +47,7 @@ public class PopupMenuVisuals {
          If following conditions are met:
           - Menu is disabled
           - Menu has caption 'Popup menu' (only applicable for linux)
-          - Menu items don't show shortcuts
+          - Menu items don't show shortcuts (except on MacOS)
```

Comment:
Is it good to restrict the test for "Windows and Linux" only ?

Reply:
It still tests for PopupMenu visuals on MacOS I suppose. We can still see if a disabled PopupMenu appears correctly. It's also a manual test. Not sure if restricting the test is actually required unless there's a bigger reason I'm missing.